app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  has_secure_password validations: false
+  validates_presence_of :email, :password, :full_name
+  validates_uniqueness_of :email
+
+end
```

Comment:
I don't understand the meaning here, you means that I should have an extra line like this?
```
class User < ActiveRecord::Base

  has_secure_password validations: false
  validates_presence_of :email, :password, :full_name
  validates_uniqueness_of :email

end

```